config/application.travis.yml

```diff
@@ -0,0 +1,5 @@
+production:
```

Comment:
Upon thinking about it, does this even need a travis version?

I would suggest an example version which may not need copying on travis deploy.